packages/x-components/src/plugins/x-plugin.ts

```diff
@@ -139,13 +139,13 @@ export class XPlugin implements PluginObject<XPluginOptions> {
    */
   protected store!: Store<any>;
   /**
-   * The global Vue, passed by the installation method. Used to apply the global mixin
+   * The Vue application instance, passed by the installation method.
    * {@link createXComponentAPIMixin}, and install the {@link https://vuex.vuejs.org/ | Vuex}
```

Comment:
Remove `x-plugin.mixin` file and also `createXComponentAPIMixin` function import here and doc